### ISSUE TITLE
[Feature request] support for HTM (Hyperscript Tagged Markup)

### ISSUE THREAD

**Issue by aMOPel** [francoismassart/eslint-plugin-tailwindcss#388]
**Is your feature request related to a problem? Please describe.**
Currently with this plugin does not trigger inside htm syntax,
even if it does trigger fine inside jsx syntax

example:
app.ts
```typescript
export function App() {
  // does not trigger on duplicate class here
  return html`<p class="m-5 m-5">route not found</p>`;
}
```

package.json:
```json
{
  "dependencies": {
    "htm": "^3.1.1",
  },
  "devDependencies": {
    "@eslint/compat": "^1.2.7",
    "@eslint/js": "^9.21.0",
    "autoprefixer": "^10.4.20",
    "eslint": "^9.21.0",
    "eslint-plugin-tailwindcss": "^3.18.0",
    "globals": "^16.0.0",
    "postcss": "^8.5.3",
    "tailwindcss": "^3.4.17",
    "typescript-eslint": "^8.24.1",
  }
}
```

eslint.config.js:
```javascript
export default [
  {
    files: ["**/*.{js,mjs,cjs,ts,jsx,tsx}"],
  },
  { languageOptions: { globals: globals.browser } },
  pluginJs.configs.recommended,
  ...tseslint.configs.recommended,
  ...tailwind.configs["flat/recommended"],
];
``` 
I also couldn't find an obvious way to make it work.
